extensions/pdh/PerformanceDataMonitor.cpp

```diff
@@ -0,0 +1,283 @@
+/**
+ * @file GenerateFlowFile.cpp
+ * GenerateFlowFile class implementation
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PerformanceDataMonitor.h"
+#include "PDHCounters.h"
+#include "MemoryConsumptionCounter.h"
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace processors {
+
+core::Relationship PerformanceDataMonitor::Success("success", "All files are routed to success");
+
+core::Property PerformanceDataMonitor::PredefinedGroups(
+    core::PropertyBuilder::createProperty("Predefined Groups")->
+    withDescription("Comma separated list to which predefined groups should be included")->
+    withDefaultValue("")->build());
+
+core::Property PerformanceDataMonitor::CustomPDHCounters(
+    core::PropertyBuilder::createProperty("Custom PDH Counters")->
+    withDescription("Comma separated list of PDHCounters to collect from")->
+    withDefaultValue("")->build());
+
+core::Property PerformanceDataMonitor::OutputFormatProperty(
+    core::PropertyBuilder::createProperty("Output format")->
```

Comment:
our other win proc, the `ConsumeWindowsEventLog`, uses `Output Format`

Reply:
changed it in  [d5959c3](https://github.com/martinzink/nifi-minifi-cpp/commit/d5959c3b1b5d0f377e7b7f2c13cc22a484b2e4a6)